shared/reducers/fs.tsx

```diff
@@ -595,6 +595,31 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
             : driverStatus
         )
       )
+    case FsGen.tlfCrStatusChanged:
+      const parsedPath = Constants.parsePath(action.payload.tlfPath)
+      const newState = action.payload.status
+      if (parsedPath.kind === Types.PathKind.Root || parsedPath.kind === Types.PathKind.TlfList) {
+        // This should not happen.
+        return state
+      }
+      return state.update('tlfs', tlfs =>
+        tlfs.update(parsedPath.tlfType, tlfList =>
+          tlfList.update(parsedPath.tlfName, tlf =>
+            tlf.update('conflict', tlfConflict => {
+              if (
+                tlfConflict.state === 'in-manual-resolution' &&
+                (newState === 'in-conflict-stuck' || newState === 'in-conflict-not-stuck')
+              ) {
+                // If the conflict is being manually resolved, ignore new
+                // conflicts that come in.
```

Comment:
If I understand correctly, the only state that InManualResolution can transit into is Finishing. After that, this TLF is deleted since all the needed data has been supposedly copied into the main one where the conflict state is cleared.

Reply:
but if we're in the middle of resolving a conflict, and then another one appears on the same TLF, what should we do?

Reply:
You mean after the TLF already branched into two? Then conflict can indeed happen again, but that'd be only on the server-view one, and it'd just be able to branch into a third TLF. The KBFS daemon would give it a different suffix in this case, so all three TLFs show up in favorites fine.

Reply:
Let's say that you have a TLF, `jakob223,songgao`. You get a conflict in it:
 - jakob223,songgao [in-conflict-stuck]
and you press "Start conflict resolution". Now you have 2 TLFs:
 - jakob223,songgao [in-manual-resolution]
 - jakob223,songgao (conflict #1) [in-manual-resolution]
and now you get a second conflict. I claim that `jakob223,songgao` should not switch from `in-manual-resolution` to `in-conflict-stuck` because that would hinder the ability of the user to finish resolving the first conflict. This amounts to forcing them to resolve the first conflict before another appears.

WDYT?

Reply:
it'd be like this:

```
jakob223,songgao
```

gets stuck

```
jakob223,songgao [conflicted]
```

hit start manual resolve

```
jakob223,songgao
jakob223,songgao (conflict #1)
```

first one (server view) get conflicted again

```
jakob223,songgao [conflicted]
jakob223,songgao (conflict#1)
```

hit manual resolve in the first one

```
jakob223,songgao
jakob223,songgao (conflict#1)
jakob223,songgao (conflict#2)
```

Note that branched out TLFs (with the conflict suffixes) can never get conflicted again -- they're a local branch and becomes readonly.

Reply:
But since we are only showing the "finish conflict resolution" button on the original / main-line TLF, we don't expose any way for the user to mark the current CR as finished in the 
```
jakob223,songgao [conflicted]
jakob223,songgao (conflict#1)
```
state. Additionally, it's not clear which conflict should be marked as finished when the user clicks "finish" in `jakob223,songgao` in the case below:
```
jakob223,songgao
jakob223,songgao (conflict#1)
jakob223,songgao (conflict#2)
```

Reply:
You are right. Not sure what the best answer is here. Perhaps "finish cr" should be in the local view(s).

Reply:
Maybe we should rename "finish" to "delete this conflict view" or something.

Reply:
Yeah, that sounds good to me too

Reply:
Cool, made this change. Thanks for all the discussion 👍